sklearn/decomposition/_pca.py

```diff
@@ -186,18 +186,18 @@ class PCA(_BasePCA):
 
         .. versionadded:: 0.18.0
 
-    tol : float >= 0, optional (default .0)
+    tol : float >= 0.0, default=0.0
```

Comment:
Can you put all the information regarding range and constraints in the description of the parameter instead of the type.
This would be necessary for all entries